### PR TITLE
System.Spatial reference cleanup

### PR DESCRIPTION
**Pull request by chipitsine**
the most confusing part actually is  tests/NuGetGallery.Core.Facts/packages.config
system.spatial was missing there, so package didn't restore at build

5.5 --> 5.6 is a cosmetic change

### CODE REVIEW

**Comment by chipitsine** [comment]
another compilation cleanup, removed 'Microsoft.Bcl.Build.targets(243,5): error : All projects referencing NuGetGallery.csproj must install nuget package Microsoft.Bcl.Build. For more information, see http://go.microsoft.com/fwlink/?LinkID=317569.' error

File: src/NuGetGallery/NuGetGallery.csproj [hunk 1]
```diff
@@ -1453,7 +1455,7 @@
           <AutoAssignPort>True</AutoAssignPort>
           <DevelopmentServerPort>80</DevelopmentServerPort>
           <DevelopmentServerVPath>/</DevelopmentServerVPath>
-          <IISUrl>http://nuget.localtest.me</IISUrl>
+          <IISUrl>http://localhost</IISUrl>
```
Comment (maartenba) [inline src/NuGetGallery/NuGetGallery.csproj]: Could you keep this one "http://nuget.localtest.me"?

Comment (chipitsine) [inline src/NuGetGallery/NuGetGallery.csproj]: Sure, I didn't mean to change IISUrl. you can leave it as "nuget.localtest.me"

**Comment by maartenba** [comment]
@yishaigalatzer We can safely merge this one. Feel free to assign to me if okay.

File: src/NuGetGallery/NuGetGallery.csproj (hunk 1 above)
Comment (chipitsine) [inline src/NuGetGallery/NuGetGallery.csproj]: can you merge it?

**Comment by yishaigalatzer** [comment]
Go for it

**Pull request merged**